pycodestyle.py

```diff
@@ -706,6 +706,8 @@ def continued_indentation(logical_line, tokens, indent_level, hang_closing,
                 # ignore token lined up with matching one from a
                 # previous line
                 pass
+            elif isinstance(visual_indent, list) and text in visual_indent:
```

Comment:
hmm so now this variable can be None, True, False, bytes, str, or a list of str?  this seems like we're trying to do too much with dynamicism here -- is there any way to make this easier to understand than overloading the type further?